src/main/kotlin/com/openlattice/indexing/BackgroundIndexingService.kt

```diff
@@ -169,7 +169,6 @@ class BackgroundIndexingService(
         return "SELECT ${ID.name}, ${LAST_WRITE.name} FROM ${IDS.name} " +
                 "WHERE ${ENTITY_SET_ID.name} = ? " +
                 "AND ${PARTITION.name} = ANY(?) " +
-                "AND ${PARTITIONS_VERSION.name} = ? " +
                 "AND $versionsClause " +
                 "AND $dirtyIdsClause "
```

Comment:
Totally unrelated, but the `dirtyIdsClause` itself should contain the `AND` prefix, otherwise if `reindex = true`, the query will fail